templates/telco/osm/index.html

```diff
@@ -237,17 +238,25 @@ <h2>OSM Community</h2>
       <ul class="p-list">
         <li class="p-list__item is-ticked">Senior Advisor: Mark Shuttleworth</li>
         <li class="p-list__item is-ticked">TSC member: Mark Beierl</li>
-        <li class="p-list__item is-ticked">RO module lead: Eduardo Sousa</li>
+        <li class="p-list__item is-ticked">Gulsum Atici: Eduardo Sousa</li>
         <li class="p-list__item is-ticked">N2VC module Lead: David Garcia</li>
         <li class="p-list__item is-ticked">VNF onboarding chair: Maciej Mazur</li>
         <li class="p-list__item is-ticked">OSM MARCOM chair: Wajeeha Hamid</li>
       </ul>
       <p>
-        <a href="https://charmed-osm.com/osm-hackfests">Discover our contributions</a>
+        <a href="https://charmed-osm.com/osm-hackfests">Discover our contributions&nbsp;&rsaquo;</a>
       </p>
     </div>
   </div>
 </section>
+<section class="p-strip is-bordered is-deep">
+  <div class="u-fixed-width">
+    <h2>Try OSM today</h2>
+    <p>Follow the <a href="https://www.youtube.com/watch?v=DJhtjnWdfi8">video</a> of installation or <a href="https://juju.is/tutorials/charmed-osm-get-started#1-introduction">tutorial</a>. If you wish to understand the whole architecture before hands on please watch this <a href="https://www.youtube.com/watch?v=3EWjL6pIXe0">video</a>.</p>
```

Comment:
The wording here is a little weird. "Follow the video of installation..." isn't technically grammatically incorrect as far as I know, but "Follow the installation video..." sounds a lot better I think. Also, "hands on" is an adjective so the sentence `If you wish to understand the whole architecture before hands on please watch this video` should be something like `If you wish to understand the whole architecture before getting hands on experience please watch this video.` I'll leave comments in the doc